src/main/java/com/prowidesoftware/swift/constraints/BicValidator.java

```diff
@@ -19,8 +19,8 @@
 import com.prowidesoftware.swift.model.BicValidationResult;
 import org.apache.commons.lang3.StringUtils;
 
-import javax.validation.ConstraintValidator;
-import javax.validation.ConstraintValidatorContext;
+import jakarta.validation.ConstraintValidator;
```

Comment:
See comment about import order above. `jakarta` should sort between `com` and `org` in this example.